test/pieces.js

```diff
@@ -464,6 +475,34 @@ describe('Pieces', function() {
     assert(page.match(/logged in/));
   });
 
+  it('can attach a tool to a person via the REST API', async function() {
+    const person1 = await apos.http.get('/api/v1/person/person1:en:published');
+    assert(person1);
+    const thing1 = await apos.http.get('/api/v1/thing/thing1:en:published');
+    assert(thing1);
+    person1._tools = [
+      {
+        ...thing1,
+        _fields: {
+          skillLevel: 5
+        }
+      }
+    ];
+    await apos.http.put('/api/v1/person/person1:en:published', {
+      body: person1,
+      jar
```

Comment:
Without using the jar here, I would have to add _tools to the `publicApiProjection` for the `person` module. Not fetching fields not in that projection is a feature rather than a bug. Threw me for a second.